prov/sm2/src/sm2.h

```diff
@@ -94,25 +91,25 @@ enum {
 };
 
 /*
- * message queue entry
  * 	next - fifo linked list next ptr
  * 		This is volatile for a reason, many things touch this
  * 		and we do not want compiler optimization here
  * 	size - Holds total size of message
  * 	cq_data - user defined CQ data
  * 	tag - used for tagged messages
+ * 	context - used for delivery complete messages
  * 	op - fi operation
- * 	op_flags - flags associated with op (ex. SM2_REMOTE_CQ_DATA, defined
- * 		   above)
+ * 	op_flags - flags associated with op
```

Comment:
Can you add a note here about only grabbing the bottom 32 bits please?

Reply:
done